app.js

```diff
@@ -11,7 +14,7 @@ app.use(express.json());
 app.get("/api", checkApi);
 app.get("/api/topics", getAllTopics);
 app.get("/api/articles", getAllArticles);
-// app.get("/api/articles/:article_id", getSpecificArticle);
+app.get("/api/articles/:article_id", getSpecificArticle);
 app.use("*", (req, res) => {
```

Comment:
Do you want `app.use` here or is there a more appropriate method that Express can use?